src/components/SocialLinks/SocialLinks.css

```diff
@@ -1,5 +1,10 @@
 .social-link {
   margin: 0px 10px;
+  transition: opacity 255ms ease;
```

Comment:
I often find anything less than 1 second to not be perceivable by users so let's bump this up to 1s :)

Reply:
In my opinion there is clearly a visible difference. Someone quickly hovering over the icon could just miss the transition assuming there is no effect.

I would suggest we bump it up to 500ms which seem to be not too long but also not too short.
https://codepen.io/anon/pen/ZPVXJo